caps/management/commands/import_emissions_data.py

```diff
@@ -24,7 +24,7 @@ def get_emissions_url() -> str:
     emissions_df = get_dataset_url(
         repo_name="la-emissions-data",
         package_name="uk_local_authority_emissions_data",
-        version_name="1",
+        version_name="1.1.0-futurecouncils",  # replace with '1' at future date
```

Comment:
I presume it's expected that this generates a "not the latest version" warning?

Reply:
Yes, I didn't give it any handling to not warn for prerelease - and tbh it's probably good it does because it'll remind us to change at some point.